### PR TITLE
Update displayimages.rst

### PR DESCRIPTION
**Pull request by haticekaratay**
### Description

This pull request resolves a minor issue with the wording in the documentation.

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/imviz/displayimages.rst [hunk 1]
```diff
@@ -411,7 +411,7 @@ To set the percentile for all the images at once:
 Colormap
 --------
 
-The spectrum of colors used to visualize data can be changed using this drop down.
+The spectrum of colors used to visualize data can be changed using this drop-down.
```
Comment (pllim) [inline docs/imviz/displayimages.rst:414]: @rosteen had a beef on "drop down" vs "dropdown" vs "drop-down" but I don't remember if I ever agreed what to use in the end.

**Review by pllim — APPROVED** [review]
Good catch! But I'll give Ricky a chance to object.

File: docs/imviz/displayimages.rst (hunk 1 above)
Comment (kecnry) [reply docs/imviz/displayimages.rst:414]: My order of preference is dropdown, drop-down, then drop down 😂

**Review by kecnry — APPROVED** [review]
Thanks!

File: docs/imviz/displayimages.rst (hunk 1 above)
Comment (rosteen) [reply docs/imviz/displayimages.rst:414]: I think my conclusion after reading up on best practices/conventions was "you shouldn't use any version of 'dropdown', just call it a menu" so it doesn't really matter 😆

**Comment by pllim** [comment]
@meeseeksdev backport to v3.7.x

**Pull request merged**